packages/generators/templates/index.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * Application entry point
+ */
+
+console.log("Hello World from <%= name %>");
```

Comment:
```suggestion
// Start writing code here
```
We can clearly guide the user here instead of a `console.log`

Reply:
I'd keep this since the purpose here is to scaffold a basic project so that users can have something working right after executing init. Another idea would be to generate a simple website, but I prefer to be less opinionated. Any suggestion on improving the generated boilerplate is welcome

Reply:
agreeing with @misterdev